apps/journeys-admin/src/components/TemplateView/TemplatePreviewTabs/TemplateCardPreview/TemplateCardPreview.tsx

```diff
@@ -149,7 +159,73 @@ export function TemplateCardPreview({
           </StyledSwiperSlide>
         )
       })}
-    </Swiper>
+      {steps.length > slidesToRender.length && (
+        <StyledSwiperSlide
+          data-testid="UseTemplatesSlide"
+          sx={{
+            width: 'unset !important',
+            cursor: 'grab',
+            zIndex: 2
+          }}
+        >
+          <Stack
+            alignItems="center"
+            justifyContent="center"
+            gap={2}
+            sx={{
+              width: { xs: 194, sm: 267 },
+              mr: { xs: 3, sm: 7 },
+              height: { xs: 295, sm: 404 },
+              borderRadius: 2,
+              backgroundColor: 'secondary.main',
+              px: 1
+            }}
+          >
+            <Typography
+              variant="overline2"
+              color="background.paper"
+              textAlign="center"
+            >
+              {t(`+${steps.length - 7} more cards`)}
```

Comment:
```suggestion
              {t('{{count}} more cards', { count: steps.length - 7 })}
```
Please take a moment to read https://docs.core.jesusfilm.org/docs/engineering-practices/internationalization/translations-tag